Breeze.UI/src/app/setup/recover/recover.component.ts

```diff
@@ -176,9 +183,13 @@ export class RecoverComponent implements OnInit {
   }
 
   private AlertIfNeeded(bitcoinErrorMessage: string, stratisErrorMessage: string) {
-        if(bitcoinErrorMessage !== "" || stratisErrorMessage !== "") {
-          let errorMessage = "<strong>Bitcoin wallet recovery:</strong><br>" + bitcoinErrorMessage + "<br><br><strong>Stratis wallet recovery:</strong><br>" + stratisErrorMessage;
-          this.genericModalService.openModal(null, errorMessage);
+        if (bitcoinErrorMessage !== '' || stratisErrorMessage !== '') {
+          const errorMessage =
```

Comment:
Definitely a step in the right direction. [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) are the tool to use
any time html templating is done from js in my opinion. white space rules will be the same as in HTML. (new line should produce a break) e.g.:
```
const errorMessage = `
<strong>Bitcoin wallet recovery:</strong>
${bitcoinErrorMessage}
<br><strong>Stratis wallet recovery:</strong>
${ stratisErrorMessage}
`;

```

Reply:
I agree, I wasn't actually changing the way it was constructed and mainly made it more readable. Pus tslint complained about the line length